src/pages/instances/InstanceList.tsx

```diff
@@ -257,7 +257,14 @@ const InstanceList: FC = () => {
         className: "u-row",
         columns: [
           {
-            content: getInstanceName(operation),
+            content: (
+              <div
+                className="u-truncate"
+                title={`Instance ${getInstanceName(operation)}`}
+              >
+                {getInstanceName(operation)}
+              </div>
+            ),
```

Comment:
I don't think you need to wrap the text in a div here. The column should have a `className` field that you can apply `u-truncate` to, would you please test that? Same story for `title`.

```suggestion
            content: getInstanceName(operation),
            className: "u-truncate",
            title: getInstanceName(operation),
```

Reply:
Hey Mason,
For continuity purposes, I am using the same approach as is seen in like 342-344 where the content itself is truncated. Do you suggest I change both implementations to the one you proposed?

Reply:
Yes thanks, that would be an improvement since we are removing unnecessary html elements and therefore improving the semantic structure of our app. I'm sure there are other places in the codebase with similar issue, but we can fix them as we come across them.